sites/layouts/documentation_pattern.html

```diff
@@ -0,0 +1,700 @@
+															]
+														}
+													]
+												}
+											]
+										}
+									]
+								},
+								{
+									"template": "[data-source-code]",
+									"test": "fn:guessType",
+									"value": "/sample",
+									"expect": "source-code",
+									"mapping": [
+										{
+											"@type": "rdf:Alt",
+											"mapping": [
+												{
+													"template": "[data-source-code-collapsed]",
+													"test": "fn:getValue",
+													"assess": "/collapsed",
+													"expect": true,
+													"mapping": [
+														{ "selector": "summary", "value": "/description" },
+														{ "selector": "code", "value": "/code", "encode": true }
+													]
+												},
+												{
+													"template": "[data-source-code-expanded]",
+													"test": "fn:getValue",
+													"assess": "/collapsed",
+													"expect": false,
+													"mapping": [
+														{ "selector": "figcaption", "value": "/description" },
+														{ "selector": "code", "value": "/code", "encode": true }
+													]
+												},
+												{
+													"template": "[data-source-code-expanded]",
+													"test": "fn:guessType",
+													"assess": "/collapsed",
+													"expect": "undefined",
+													"mapping": [
+														{ "selector": "figcaption", "value": "/description" },
+														{ "selector": "code", "value": "/code", "encode": true }
+													]
+												}
+											]
+										}
+									]
+								},
+								{
+									"template": "[data-code-compare]",
+									"test": "fn:guessType",
+									"value": "/sample",
+									"expect": "code-compare",
+									"mapping": [
+										{ "selector": "figcaption[data-rev]", "value": "/rev_title" },
+										{ "selector": "code[data-rev]", "value": "/rev_code", "isHTML": true },
+										{ "selector": "figcaption[data-from]", "value": "/from_title" },
+										{ "selector": "code[data-from]", "value": "/from_code", "isHTML": true }
+									]
+								}
+
+							]
+						}
+					]
+				}
+			]
+		},
+
+		{ "selector": "[data-history]", "value": "/history",
+			"mapping": [
+				{ "selector": "li", "value": "/{{lng}}" }
+			]
+		}
+
+	],
+	"trigger": true
+}'>
+
+	<template>
+		<h3>
+			<span data-name>[[ Name of the variation ]]</span>
+			<template data-status-stable>
+				<span class="label label-success mrgn-lft-sm"><span class="wb-inv">Status: </span>Stable</span>
+			</template>
+			<template data-status-tobereviewed>
+				<span class="label label-info mrgn-lft-sm"><span class="wb-inv">Status: </span>To be reviewed</span>
+			</template>
+			<template data-status-provisional>
+				<span class="label label-warning mrgn-lft-sm"><span class="wb-inv">Status: </span>Provisional</span>
+			</template>
+			<template data-status-deprecated>
+				<span class="label label-default mrgn-lft-sm"><span class="wb-inv">Status: </span>Deprecated</span>
+			</template>
```

Comment:
There is no stability indicator because a design pattern out of scope of our versioning api

```suggestion
```